src/Forms/NumericField.php

```diff
@@ -154,6 +154,9 @@ public function setValue($value, $data = null)
      */
     protected function cast($value)
     {
+        if (strlen($value) === 0) {
+            return false;
```

Comment:
Let's try null and see if we get better value for money :) It feels more semantically correct than false for empty numeric values.